options.js

```diff
@@ -0,0 +1,83 @@
+const options = {
+  compatibilityMode: false,
+  copyButtons: true,
+  correctnewlines: false,
+  markdownOptions: {
+    fences: true,
+    commonmark: true,
+    gfm: true,
+    ruleSpaces: false,
+    listItemIndent: '1',
+    spacedTable: true,
+    paddedTable: true,
+    setext: true,
+  },
+  normalize: true,
+  settings: {
+    position: false,
+  },
+};
+
+// NOTE: disabling newline, paragraph, or text trips remark into an infinite loop!
+const blocks = [
+  // 'newline',
+  'indentedCode',
+  'fencedCode',
+  'blockquote',
+  'atxHeading',
+  'thematicBreak',
+  'list',
+  'setextHeading',
+  'html',
+  'footnote',
+  'definition',
+  'table',
+  // 'paragraph',
+];
+
+const inlines = [
+  'escape',
+  'autoLink',
+  'url',
+  'html',
+  'link',
+  'reference',
+  'strong',
+  'emphasis',
+  'deletion',
+  'code',
+  'break',
+  // 'text',
+];
+
+const toBeDecorated = {
+  inlines: inlines.filter(i => !['link', 'reference'].includes(i)),
+  blocks: [],
+};
+
+const disableTokenizers = {
+  blocks: {
+    disableTokenizers: {
+      inline: toBeDecorated.inlines,
+      block: toBeDecorated.blocks,
+    },
+  },
+  inlines: {
+    disableTokenizers: {
+      inline: inlines.filter(i => !toBeDecorated.inlines.includes(i)),
+      block: blocks.filter(b => !toBeDecorated.blocks.includes(b)),
+    },
+  },
+};
+
+const parseOptions = (opts = {}) => {
+  if (opts.tokenizerSet in disableTokenizers) {
+    return { ...disableTokenizers[opts.tokenizerSet], ...opts };
+  } else if (opts.tokenizerSet) {
+    throw new Error(`opts.tokenizerSet "${opts.tokenizerSet}" not one of "${Object.keys(disableTokenizers)}"`);
+  }
```

Comment:
Can we rename the public-facing option to `disableTokenizers`? It feels a bit redundant here, but I think it'd be a bit more descriptive of what it actually does for the end user.
```suggestion
  if (opts.disableTokenizers in disableTokenizers) {
    return { ...disableTokenizers[opts.disableTokenizers], ...opts };
  } else if (opts.disableTokenizers) {
    throw new Error(`opts.disableTokenizers "${opts.disableTokenizers}" not one of "${Object.keys(disableTokenizers)}"`);
  }
```